styles/pymakr.less

```diff
@@ -2,11 +2,14 @@
 //
```

Comment:
I think in the future we should not use just one less file for all styles.

Isn't it possible to use CSS modules? that would be nice.

Reply:
Yes, it's possible. I just kept the original way that was done. I agree that would be a nice idea.

Reply:
Yes I agree with you. I just kept using the same way it was implemented.

Reply:
Yes I agree with you. I just kept using the same way it was implemented.